### ISSUE TITLE
Index out of range on Decode token

### ISSUE THREAD

**Issue by WendelHime** [gbrlsnchs/jwt#25]
Hi, I'm trying to decode the token but it throws `index out of range`

```bash
panic: runtime error: index out of range

goroutine 9 [running]:
github.com/gbrlsnchs/jwt/v3.(*Audience).UnmarshalJSON(0xc0001216a0, 0xc00006616e, 0xf, 0x43, 0x7fa7fb14c158, 0xc0001216a0)
	/go/pkg/mod/github.com/gbrlsnchs/jwt/v3@v3.0.0-alpha.2/audience.go:39 +0x1df
encoding/json.(*decodeState).array(0xc00016f680, 0x930ec0, 0xc0001216a0, 0x197, 0xc00016f6a8, 0x5b)
	/usr/local/go/src/encoding/json/decode.go:505 +0xc6d
encoding/json.(*decodeState).value(0xc00016f680, 0x930ec0, 0xc0001216a0, 0x197, 0x930ec0, 0xc0001216a0)
	/usr/local/go/src/encoding/json/decode.go:371 +0xff
encoding/json.(*decodeState).object(0xc00016f680, 0x929120, 0xc000121680, 0x16, 0xc00016f6a8, 0xc00018c87b)
	/usr/local/go/src/encoding/json/decode.go:763 +0x1899
encoding/json.(*decodeState).value(0xc00016f680, 0x929120, 0xc000121680, 0x16, 0xc0002176c0, 0x52fd27)
	/usr/local/go/src/encoding/json/decode.go:381 +0x6e
encoding/json.(*decodeState).unmarshal(0xc00016f680, 0x929120, 0xc000121680, 0xc00016f6a8, 0x0)
	/usr/local/go/src/encoding/json/decode.go:179 +0x1fe
encoding/json.Unmarshal(0xc000066150, 0x61, 0x61, 0x929120, 0xc000121680, 0x82, 0xb9)
	/usr/local/go/src/encoding/json/decode.go:106 +0x123
github.com/gbrlsnchs/jwt/v3.RawToken.Decode(0xc00037c5a0, 0xe5, 0xf0, 0x36, 0xb9, 0xc000065c80, 0x929120, 0xc000121680, 0x0, 0x0)
	/go/pkg/mod/github.com/gbrlsnchs/jwt/v3@v3.0.0-alpha.2/raw_token.go:65 +0x327
```

I'm using the default jwt.Payload and the v3 branch

Token example:

```
eyJhbGciOiJIUzI1NiIsImtpZCI6ImF1dGgiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhdXRoIiwic3ViIjoiMSIsImF1ZCI6WyJhZG0iXSwiZXhwIjoxNTgyOTk5OTg5LCJuYmYiOjE1NTE4OTc3ODksImlhdCI6MTU1MTg5NTk4OX0.SGsv7DBtpjkJZeRaikcWO48UXG7aPRpmIEPMesQQVqQ
```

**Comment by gbrlsnchs** [comment]
I will release a new version in a little while that fixes this issue. `master` already has the fix if you're in a hurry.

**Comment by gbrlsnchs** [comment]
Fixed in [`v3.0.0-alpha.3`](https://github.com/gbrlsnchs/jwt/tree/v3.0.0-alpha.3). :smile:

**Comment by WendelHime** [comment]
Thanks @gbrlsnchs !

I think the README is missing something related to how to access this tagged versions (such as the environment variable GO111MODULE=on)
